ci/cloudbuild/build.sh

```diff
@@ -242,7 +242,9 @@ if [[ "${DOCKER_FLAG}" = "true" ]]; then
   )
   cmd=(ci/cloudbuild/build.sh --local "${BUILD_NAME}")
   if [[ "${SHELL_FLAG}" = "true" ]]; then
-    printf "To run the build manually:\n  %s\n\n" "${cmd[*]}"
+    printf "To run the build manually:\n  "
```

Comment:
Nit: Previously you indented the command by 2, but this will result in 3.

Reply:
I know. 3 looked nicer.